### PR TITLE
Style polishing

### PR DESCRIPTION
**Pull request by timkeresey**
### What does this PR do?

- adds cosmetic changes to various components to improve UI/UX.
- Fixes spelling errors

### Where should the reviewer start?

- components/Home
- components/Journey
- components/Question

### How should this be manually tested?

-By navigating through the app in the browser

### What are the relevant tickets?

- #115

### CODE REVIEW

**Review by BlakeDonn — APPROVED** [review]
This nicely cleans up a few things I have been noticing also, thanks for that.

File: src/components/Home/Home.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const Home: React.FC = () => {
             Start your journey with us
           </h1>
           <h1 className="plan-title" data-testid="Dream Home-3">
-            to start building your path to your dream home...
+            building a path to your dream home!
```
Comment (Atos20) [inline src/components/Home/Home.tsx:50]: Thanks for looking at the punctuation and grammar.

File: src/components/Question/Question.tsx [hunk 2]
```diff
@@ -101,10 +101,10 @@ export const Question: React.FC<Props> = ({ updateAllAnswers }) => {
               <h1 data-testid="description-title" className="question-desc">
                 Your Goal Home Price or Your Rent
               </h1>
-              
-              <h4 className="resource-2">If you have a number in mind of how much you'd like your Dream Home to cost, select "Goal Principle." If you don't, that's ok! Select "Rent" and we'll use what you're currently paying in rent to calculate a potential path for you! </h4>
+
+              <h4 className="resource-2">If you have a number in mind of how much you'd like your Dream Home to cost, select "Goal Principal." If you don't, that's ok! Select "Rent" and we'll use what you're currently paying in rent to calculate a potential path for you! </h4>
```
Comment (Atos20) [inline src/components/Question/Question.tsx:105]: Thanks for changing principle to principal. I will never forget this lol

**Review by Atos20 — COMMENTED** [review]
Thanks for the adjustments 👍🏽

**Pull request merged**